utils/misc_test.go

```diff
@@ -160,3 +160,43 @@ func (s *TestSuite) TestRandomID(c *C) {
 		c.Assert(len(result), Equals, testCase.expectedLength, Commentf(test.ErrResultFmt, testName))
 	}
 }
+
+func (s TestSuite) TestGenRandNumber(c *C) {
+	type testCase struct {
+		inputLower int64
+		inputUpper int64
+
+		expect_success bool
+		ret_range      []int64
+	}
+	testCases := map[string]testCase{
+		"GenRandNumber(...)": {
```

Comment:
```suggestion
		"GenerateRandomNumber(...)": {
```
This also applies to the following cases.